.ci/azure-pipelines-test.yml

```diff
@@ -28,12 +28,26 @@ jobs:
         submodules: true
         persistCredentials: false
 
+      # This is required for the SonarCloud analyzer
+      - task: UseDotNet@2
+        displayName: "Install .NET Core SDK 2.1"
+        inputs:
+          packageType: sdk
+          version: '2.1.805'
```

Comment:
FYI I decided to use 2.1 here instead of 2.2 because 2.2 is in End Of Life status but 2.1 is a LTS release: https://dotnet.microsoft.com/download/dotnet-core

Reply:
Sound good.